R/draws-index.R

```diff
@@ -141,6 +141,37 @@ remove_reserved_variable_names <- function(variables, reserved) {
   x
 }
 
+#' Set variable names in `draws` objects
+#'
+#' Set variable names for all variables in a [`draws`] object. Useful
+#' when using pipe operators.
+#'
+#' @param x (draws) A [`draws`] object.
+#' @param variables (character) new variable names.
+#' @template args-methods-dots
+#'
+#' @return Returns a [`draws`] object of the same format as `x`, with
+#'   variables named as specified.
+#'
+#' @seealso [`variables`]
+#' @examples
+#' x <- as_draws(matrix(rnorm(100), ncol = 2))
+#' variables(x)
+#'
+#' set_variables(x, c("theta[1]", "theta[2]"))
```

Comment:
you are not assigning to x here. Is that intentional?

Reply:
Good catch! Fixed this now.